hugo/layouts/partials/copy_to_clipboard.html

```diff
@@ -0,0 +1,4 @@
+{{ if (findRE "<pre" .Content 1) }}
```

Comment:
For an alternative approach to detect copy-blocks, see [render hook examples](https://gohugo.io/render-hooks/code-blocks/#examples). The idea is to set `$.Page.Store.Set "needs_clipboard" true` in the shortcode template, and then use `{{ if $.Page.Store.Get "needs_clipboard" }}` Not 100% sure if what works for render hooks will work for shortcodes though.

Reply:
I've made some progress with render hooks now, I'll look into this bit now!

Reply:
Thanks for the link. Yes I can make this work and remove the need for the file copy_to_clipboard. A small downside was needing to add a new if block to `hugo/layouts/_default/baseof.html`, but I don't think that does much harm overall. Added this in commit 1bb8a7d3249a65dce7432f659ee8f3595d51d299